src/index.d.ts

```diff
@@ -0,0 +1,99 @@
+declare module "react-dropzone-s3-uploader" {
+  import { Component } from "react";
+  import { ReactS3UploaderProps as OriginalReactS3UploaderProps } from "react-s3-uploader";
+
+  // The type from the dependency is not correct, so we need to fix part of it to be able to use it.
+  export interface FixedReactS3UploaderProps
```

Comment:
Other than this type addition, this file was pulled from `exec_online:components/enterprise_portal/client/types/react-dropzone-s3-uploader.d.ts`